### ISSUE TITLE
Error 500 when using PeriodInsertPlugin

### ISSUE THREAD

**Issue by MBe-iUS** [lnNgyn1/PeriodInsertBundle#5]
Hi,

thanks for having updated this Plugin to Kimai2 v2 (or so)!

I encountered an error while using it: 

```
kimai-1  | localhost:8001 192.168.10.5 - - [22/Oct/2024:18:09:14 +0200] "GET /api/projects?customer=8&visible=1&start=&end= HTTP/1.1" 200 3716 "http://192.168.40.102/de/period_insert" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:128.0) Gecko/20100101 Firefox/128.0"
kimai-1  | localhost:8001 192.168.10.5 - - [22/Oct/2024:18:09:14 +0200] "GET /api/activities?globals=true&visible=1 HTTP/1.1" 200 1463 "http://192.168.40.102/de/period_insert" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:128.0) Gecko/20100101 Firefox/128.0"
kimai-1  | localhost:8001 192.168.10.5 - - [22/Oct/2024:18:09:17 +0200] "GET /api/activities?project=8&visible=1 HTTP/1.1" 200 4351 "http://192.168.40.102/de/period_insert" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:128.0) Gecko/20100101 Firefox/128.0"
kimai-1  | [2024-10-22T16:09:44.998870+00:00] request.INFO: Matched route "period_insert". {"route":"period_insert","route_parameters":{"_route":"period_insert","_controller":"KimaiPlugin\\PeriodInsertBundle\\Controller\\PeriodInsertController::indexAction","_locale":"de"},"request_uri":"http://192.168.40.102/de/period_insert","method":"POST"} {"channel":"request"}
kimai-1  | [2024-10-22T16:09:45.155723+00:00] request.CRITICAL: Uncaught PHP Exception Error: "Call to a member function format() on null" at PeriodInsert.php line 123 {"exception":"[object] (Error(code: 0): Call to a member function format() on null at /opt/kimai/var/plugins/PeriodInsertBundle/Entity/PeriodInsert.php:123)"} {"channel":"request"}
kimai-1  | localhost:8001 192.168.10.5 - - [22/Oct/2024:18:09:44 +0200] "POST /de/period_insert HTTP/1.1" 500 1917 "http://192.168.40.102/de/period_insert" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:128.0) Gecko/20100101 Firefox/128.0"
```

Any idea, what it could be? Happy to supply more info.

It is running in a Docker, Kimai 2.24.0, PeriodInsertPlugin v1.2 (it says 1.0 in the backend)

Thanks!

**Comment by lnNgyn1** [comment]
Hi, thanks for opening this issue! I think I know exactly what's causing the issue. I will provide a fix for it shortly (and update the plugin version in the `composer.json` file).

**Comment by lnNgyn1** [comment]
Also, what time-tracking mode are you currently using? There are three different modes: `Default`, `Duration`, and `Time-clock`. I think the bug you came across only applies to the `Duration` mode.

**Comment by MBe-iUS** [comment]
Duration it is.

**Comment by lnNgyn1** [comment]
Okay, commit aa0ab82 should fix your bug. I re-released version 1.2, so you can install the latest release. Let me know if you run into any other issues!

**Comment by MBe-iUS** [comment]
That solved it! Thank you!
